rdmd.d

```diff
@@ -692,6 +692,27 @@ private string[string] getDependencies(string rootModule, string workDir,
 
     immutable rootDir = dirName(rootModule);
 
+    // In most cases we don't have any dependencies except the standard library
+    auto file = readText(rootModule);
+    bool needInspection = false;
+    outer: foreach (match; file.matchAll(ctRegex!`import\s+(.*);`))
+    {
+        foreach (mod; match[1].splitter(","))
+        {
+            if (!(mod.startsWith("std") || mod.startsWith("std")))
```

Comment:
I'm guessing you meant to write `if (!(mod.startsWith("std") || mod.startsWith("core")))`